packages/front-end/components/Settings/EditDataSource/DimensionMetadata/UpdateDimensionMetadata.tsx

```diff
@@ -231,64 +284,37 @@ export const DimensionSlicesRunner: FC<DimensionSlicesRunnerProps> = ({
                 </div>
               </div>
             ) : null}
+            <div className="flex-1" />
```

Comment:
Is this needed? Seems like it might not be doing anything since its self closing

Reply:
Yes, this is how fill space in a couple of places so that the "days to look back" div stays right aligned